src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1012,22 +1012,22 @@
     <value>Processing XML from untrusted data may load dangerous external references, which should be restricted by using an XmlReader with a secure resolver or with DTD processing disabled.</value>
   </data>
   <data name="UseXmlReaderForDataSetReadXml" xml:space="preserve">
-    <value>Use XmlReader For DataSet Read Xml</value>
+    <value>Use XmlReader for DataSet.ReadXml()</value>
```

Comment:
Our strings are not consistent, but ideally all API references should use single quotes around them.